src/PerfView/Dialogs/RunCommandDialog.xaml.cs

```diff
@@ -64,11 +64,18 @@ public RunCommandDialog(CommandLineArgs args, MainWindow mainWindow, bool isColl
             m_mainWindow = mainWindow;
 
             CurrentDirTextBox.Text = Environment.CurrentDirectory;
+
+            // Initialize the CommandToRun history if available. 
+            var commandToRunHistory = App.ConfigData["CommandToRunHistory"];
+            if (commandToRunHistory != null)
+                CommandToRunTextBox.SetHistory(commandToRunHistory.Split(';'));
```

Comment:
I suppose I could change the name of `CommandToRunTextBox` to `CommandToRunHistoryComboBox` (which seems quite verbose), or just `CommandToRunControl`. Comments?